src/epivizFileParser/sceTileDB.py

```diff
@@ -0,0 +1,114 @@
+import tiledb
+import numpy as np
+import pandas as pd
+
+from .utils import dense_to_sparse
+
+__author__ = "Jayaram Kancherla, Victor Alexandru"
+__copyright__ = "jkanche, alexanv8"
+__license__ = "mit"
+
+
+class sceTileDB:
+    """
+    TileDB Class to parse single-cell folders.
+    Args:
+        path (str): local full path to an experiment inside a dataset.
+            ex: tests/data/single-cell/GEPIVIZ_DS000010065/part_1
```

Comment:
remove

Reply:
document the overall structure of the directory

Reply:
fixed